syntaxes/z80-macroasm.tmLanguage.json

```diff
@@ -70,6 +70,10 @@
 					"name": "keyword.control.z80asm",
 					"match": "(?i:(?<=\\s)(?:amsdos|breakpoint|buildcpr|buildsna|run|save|setcpc|setcrtc)\\b)"
 				},
+				{
+					"name": "keyword.control.z80asm",
+					"match": "(?i:(?<=\\s)(?:{hex}|{hex[248]}|{bin}|{bin8}|{bin16}|{bin32}|{int}|{eval}|{})\\b)"
```

Comment:
If it should match only after `print` keyword then it should be stated in that positive lookahead at the start. Also whole matching group with alternatives could be done much nicer:
`(?i:(?<=print\\s)\\{(?:hex[248]?|bin(?:8|16|32)?|int|eval)?\\}\\b)`

And finally I suggest to move this selector at the end of `"mnemonics"` section and put into `constant.language.operator.z80asm` highlight group.

Reply:
I will propose a rewrite. However not exactly similar to that, because:
- `print` is not the only directive that can accept such modifier
- we can have more than one modifier per line
No problem if my proposal will need to be patched if it can be done better

Reply:
It's okay, there we could use enumeration in the positive lookahead and plus-quantifier to match one and more "curly directives", for example:
`(?i:(?<=(print|fail|foo|bar)\\s)(?:\\{(?:hex[248]?|bin(?:8|16|32)?|int|eval)?\\})+\\b)`

Reply:
It seems this regex is incorrect. By using it, I have no more syntaxic coloration on z80 code. I think my visual studio code is unable to provide me the error message because I have something badly installed (it always raises an error related to `npm: watch` when I press `F5`).

Anyway, I m' neither a regex master and unable to find the mistake.

To be clear,  here is a real world usage
```z80
 print  "ptr_screen_line_{idx_code}) = ", {hex}memory_screen_line_{idx_screen}, '-', {hex}(memory_screen_line_{idx_screen} +screen_width )
```